webview/src/experiments/components/table/CellSecondaryName.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { VSCodeTag } from '@vscode/webview-ui-toolkit/react'
+import { CommitData } from 'dvc/src/experiments/webview/contract'
+import styles from './styles.module.scss'
+import Tooltip from '../../../shared/components/tooltip/Tooltip'
+
+export const CellSecondaryName: React.FC<{
+  displayNameOrParent: string
+  commit?: CommitData
+  sha?: string
+}> = ({ displayNameOrParent, commit, sha }) => {
+  const children = (
+    <span className={styles.experimentCellSecondaryName}>
+      {displayNameOrParent}
+    </span>
+  )
+  if (!commit) {
+    return children
+  }
+
+  const { tags, author, message, date } = commit
+  const tooltipContent = (
+    <div className={styles.experimentCellSecondaryNameTooltip}>
+      <div>
+        <p className={styles.sha}>{sha?.slice(0, 7)}</p>
+        {tags.length > 0 && (
+          <>
+            {tags.map(tag => (
+              <VSCodeTag className={styles.tag} key={tag}>
+                {tag}
+              </VSCodeTag>
+            ))}
+          </>
+        )}
+      </div>
+      <p>
+        {author}, {date}
+      </p>
+      <p className={styles.message}>{message}</p>
+    </div>
+  )
+
+  return (
+    <Tooltip
+      placement="bottom-start"
+      appendTo={document.body}
+      content={tooltipContent}
```

Comment:
Originally I was going to make the tooltip interactive but ran into a bug when you try to click on the text. Should be fixable but figured it could be done in a follow-up: 

https://user-images.githubusercontent.com/43496356/212735822-dae8a6c2-a85c-4436-897b-d334a121b968.mov

Reply:
Was this with the `interactive` option set to `true` on the tooltip?

Reply:
> Was this with the interactive option set to true on the tooltip?

Yes!